app/js/config/environment/staging.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  auth_url: 'http://api-staging.andela.com/login',
   slackZhishiChannel: 'https://hooks.slack.com/services/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',
```

Comment:
seems keys in `staging` are a bit different from `dev` and `prod`, especially `slack`, could you update this? It's not related to your changes either, but we want to be absolutely sure that if something works in dev, we can test in staging and it works in prod